curp/tests/common/curp_group.rs

```diff
@@ -79,34 +78,25 @@ impl CurpGroup {
 
                 let cluster_info = Arc::new(ClusterMember::new(all.clone(), id.clone()));
 
-                let rt = tokio::runtime::Builder::new_multi_thread()
-                    .worker_threads(2)
-                    .thread_name(format!("rt-{id}"))
-                    .enable_all()
-                    .build()
-                    .unwrap();
-                let handle = rt.handle().clone();
-
                 let storage_path_c = storage_path.clone();
                 let role_change_cb = TestRoleChange::default();
                 let role_change_arc = role_change_cb.get_inner_arc();
-                thread::spawn(move || {
-                    handle.spawn(Rpc::run_from_listener(
-                        cluster_info,
-                        i == 0,
-                        listener,
-                        ce,
-                        Box::new(MemorySnapshotAllocator),
-                        role_change_cb,
-                        Arc::new(
-                            CurpConfigBuilder::default()
-                                .data_dir(storage_path_c)
-                                .log_entries_cap(10)
-                                .build()
-                                .unwrap(),
-                        ),
-                    ));
-                });
+
+                tokio::spawn(Rpc::run_from_listener(
```

Comment:
Why do you remove the runtime here? I guess using runtime here is for renaming a tokio 
 thread which may be beneficial to debug.